### PR TITLE
[F1] Executing Launcher with new arguments (3rd)

### PR DESCRIPTION
**Pull request by stjohnjohnson**
[Feature](https://github.com/screwdriver-cd/screwdriver/issues/77)

Passing:
 - buildId - Build Identifier (to look up stuff)
 - container - Container to start in
 - apiUri - URI to hit our API
 - token - Token to be able to read/write from the API

Blocked on https://github.com/screwdriver-cd/executor-base/pull/13

@jer I need the expected version of the JobTools that will contain this new launcher

### CODE REVIEW

**Comment by jer** [comment]
Seems good! The launcher should be called like this though:

/opt/screwdriver/launch --api-url {{api_uri}} {{build_id}}

with $SD_TOKEN set to the token value.

**Comment by stjohnjohnson** [comment]
@jer updated

File: test/index.test.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ describe('index', () => {
 
         fsMock.readFileSync.withArgs('/etc/kubernetes/apikey/token').returns('api_key');
         fsMock.readFileSync.withArgs(sinon.match(/config\/job.yaml.tim/))
-        .returns(TEST_TIM_YAML);
+          .returns(TEST_TIM_YAML);
```
Comment (FenrirUnbound) [inline test/index.test.js]: 4 spaces?

**Comment by d2lam** [comment]
Please also update README to reflect the change in `start`
https://github.com/screwdriver-cd/executor-k8s/blob/master/README.md#start

File: index.js [hunk 2]
```diff
@@ -7,25 +7,23 @@ const request = require('request');
...
+     * @param  {String} options.token     Api Token to make requests with
+     * @param  {String} options.host      Kubernetes hostname to make requests to
+     * @param  {String} options.version   Job Tools container version to use
      */
     constructor(options) {
         super();
 
         this.token = options.token;
         this.host = options.host;
+        this.version = options.version;
```
Comment (nkatzman) [inline index.js]: default to latest?

**Comment by nkatzman** [comment]
👍

**Pull request merged**